triggers/cron.py

```diff
@@ -49,7 +50,10 @@ def stop(self):
     def _cron(self):
         """ Called every minute to check if cron job should notify signals """
         self.logger.debug("Checking if cron emit should run")
-        now = datetime.utcnow()
+        if (self.utc()):
+            now = datetime.utcnow()
+        else:
+            now = datetime.now()
```

Comment:
This looks good but before it can get merged in you gotta do the fun part and add a test function inside test/triggers/test_cron.py 😺